### DISCUSSION TITLE
Semi realistic earth reentry

### DISCUSSION THREAD

**Discussion by JustPyrrha** [TeamGalacticraft/Galacticraft#172] [Feature Requests]
Currently Galacticraft has the player land back on earth using a parachute and parachest. It would be cool to have the player land in a capsule.

**Comment by vos6434** [comment]
or a tesla.

**Comment by Atakku** [comment]
Like this?
![](https://media.giphy.com/media/YOAXX5bY2MIEK1hWwb/giphy-downsized.gif)

**Comment by craftablescience** [comment]
No, that's the first stage landing, it would be the second stage coming back.

**Comment by EzerArch** [comment]
> It would be cool to have the player land in a capsule.

Like the 1st landing in Venus? Yes!

**Comment by Asteroid4** [comment]
I can just imagine seeing the capsule heating up when entering the atmosphere.
